tests/EventTest.php

```diff
@@ -119,32 +119,27 @@ public function testToArray(): void
         $this->assertEquals($expected, $event->toArray());
     }
 
-    public function testToArrayWithMessage(): void
+    /**
+     * @dataProvider toArrayWithMessageDataProvider
+     */
+    public function testToArrayWithMessage(array $message, $expectedValue): void
     {
         $event = new Event();
-        $event->setMessage('foo bar');
+        call_user_func_array([$event, 'setMessage'], $message);
 
         $data = $event->toArray();
 
         $this->assertArrayHasKey('message', $data);
-        $this->assertEquals('foo bar', $data['message']);
+        $this->assertSame($expectedValue, $data['message']);
     }
 
-    public function testToArrayWithMessageWithParams(): void
+    public function toArrayWithMessageDataProvider(): array
     {
-        $expected = [
-            'message' => 'foo %s',
-            'params' => ['bar'],
-            'formatted' => 'foo bar',
+        return [
+            [['foo bar'], 'foo bar'],
```

Comment:
Please split this array into multiple lines (have a look at how other data providers are formatted)

Reply:
It looks like some data provider arrays are split into multiple lines while some are not.  Cleaning all of those up seems out of scope for this PR though.

Reply:
I meant the data provider of this file. You should format the arrays one item per line if possible, e.g.

```php
[
    [
        'foo @bar',
        [
            '@bar' => 'bar',
        ],
        'foo bar',
    ],
    [
        'message' => 'foo @bar',
        'params' => [
            '@bar' => 'bar',
        ],
        'formatted' => 'foo bar',
    ],
]
```

Reply:
FYI there was a pre-existing data provider in this file that has "packed" array syntax:
```
            ['modules', ['foo' => '0.0.1', 'bar' => '0.0.2'], ['modules' => ['foo' => '0.0.1', 'bar' => '0.0.2']]],
```

Reply:
Thank you for reporting this, some inconsistencies are indeed possible since style is not forced by PHPCS-Fixer. However I would prefer usage of the "expanded" version